build.sbt

```diff
@@ -2348,10 +2348,18 @@ lazy val `engine-runner` = project
             "-Dnic=nic"
           ),
           mainClass = Some("org.enso.runner.Main"),
-          additionalCp = Seq(
-            "runtime.jar",
-            "runner.jar"
-          ),
+          additionalCp = () => {
```

Comment:
I believe `additionalCp` does not need to be a function. You can just write something like:
```scala
additionalCp = {
  val core = Seq(...)
  val jars = Seq(...)
  core ++ jars
}
```
that will assign a proper sequence to `additionalCp`.

Reply:
I don't think this works. I need to execute `buildEngineDistribution` first and only then search for the JAR files. Otherwise (with a clean build) they will not be present yet when the `{ ... }` code runs. with `() => { ... }` the code gets executed after the JARs are generated.